packages/rancher-monitoring/rancher-monitoring/generated-changes/patch/values.yaml.patch

```diff
@@ -231,12 +248,25 @@
 +        operator: "Exists"
 +      - effect: "NoSchedule"
 +        operator: "Exists"
++  kubeVersionOverrides:
++  - constraint: "< 1.22"
++    values:
++      metricsPort: 10252 # default to insecure port in k8s < 1.22
++      clients:
++        https:
++          enabled: false
++          insecureSkipVerify: false
++          useServiceAccountCredentials: false
 +
 +rke2Scheduler:
```

Comment:
Q: why are we modifying only kubeScheduler in RKE2 and not RKE1?

A: RKE1 has not moved to using the secure port yet for scheduler, only for controller manager. When RKE1 moves to using the scheduler port in a future k8s release (probably 1.23), we will need to fix this.